.woodpecker.yml

```diff
@@ -216,11 +218,11 @@ pipeline:
 
   # using https://github.com/pksunkara/cargo-workspaces
   publish_to_crates_io:
-    image: clux/muslrust:1.67.0
+    image: *muslrust_image
     commands:
       - cargo install cargo-workspaces
       - cp -r migrations crates/db_schema/
-      - cargo login "$CARGO_TOKEN"
+      - cargo login "$CARGO_API_TOKEN"
```

Comment:
make sure the secret is called "cargo_api_token" or "CARGO_API_TOKEN" in woodpecker too

Reply:
Checked, that's good.